package.json

```diff
@@ -26,13 +26,13 @@
   },
   "homepage": "https://github.com/ethereum-alarm-clock/eac.js-cli#readme",
   "dependencies": {
+    "@ethereum-alarm-clock/timenode-core": "github:ethereum-alarm-clock/timenode-core#optional-claiming",
```

Comment:
probably redirect this to the main timenode-core.

Reply:
Just merged the last, will release a new version and we can change this to 4.0.4

Reply:
Waiting for the npm release.